src/DependencyInjection/SonataPageExtension.php

```diff
@@ -517,4 +530,109 @@ public function configurePageServices(ContainerBuilder $container, array $config
         $definition = $container->getDefinition('sonata.page.page_service_manager');
         $definition->addMethodCall('setDefault', [new Reference($config['default_page_service'])]);
     }
+
+    private function registerSonataDoctrineMapping(array $config): void
+    {
+        if (!class_exists($config['class']['page'])) {
+            return;
+        }
+
+        $collector = DoctrineCollector::getInstance();
+
+        $pageChildrenOptions = OptionsBuilder::createOneToMany('children', $config['class']['page'])
```

Comment:
Is there a reason for this OTVs? Can you please inline them.

Reply:
Inlined.